src/gui/embed.cpp

```diff
@@ -45,73 +45,70 @@ namespace
 #include "embedded_resources.h"
 
 
-QPixmap getIconPixmap( const char * _name, int _w, int _h )
+QPixmap getIconPixmap( const char * pixmapName, int width, int height )
```

Comment:
I see all sorts of acronyms in the code that I'm having trouble understanding. I'd rather not use one-letter variable names whenever possible, but you're right in that it's probably quite clear as `w`/`h` as well.